CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.7.0 (September 21, 2020)
+
+Since `1.7.0`, vscode-quarkus becomes an extension of [vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile), see [CHANGELOG of vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile/CHANGELOG.md#010-september-14-2020)
+
+### Enhancements
+
+ * Notification when propertiesLanguageMismatch is forceQuarkus and user tries to set languageId to something else. See [#285](https://github.com/redhat-developer/vscode-quarkus/issues/285).
```

Comment:
when the `quarkus.tools.propertiesLanguageMismatch` setting is set to `forceQuarkus`,

Reply:
updated